src/app/shared/documentation-items/documentation-items.ts

```diff
@@ -29,7 +29,7 @@ const DOCS = [
   },
   {
     id: 'nav',
-    name: 'Navigation',
+    name: 'Navigation Components',
```

Comment:
Ah, I just added "Components" to it since the screen reader otherwise reads it as "Navigation navigation". I'll see if we can change it to an aria label instead. 👍